### PR TITLE
Improvements for the NN implementation

### PR DESCRIPTION
**Pull request by volpatto**
This PR adds more flexibility to use and set different NN architectures, as well as a better API to the NN model, exposing more parameters for different NN configurations.

### CODE REVIEW

File: src/fpnn/model_training.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 from sklearn.metrics import r2_score
 from sklearn.preprocessing import StandardScaler
-from tensorflow import keras
+from tensorflow import keras, random, data
 from keras import layers, models, utils
 
 
 # Set a random seed
 utils.set_random_seed(42)
+random.set_seed(42)
```
Comment (volpatto) [inline src/fpnn/model_training.py:15]: - [ ] Define a var for the seed. Useful when spreading the same seed value for different seed setters.

**Review by Mauri-pos — APPROVED** [review]
